_test/outputxmlhelper.test.php

```diff
@@ -83,4 +92,71 @@ public function parameterProviderForXMLCallWithFloatParams()
             'invalid start = 1.1 and count = 02.00' => [1.1, 02.00]
         ];
     }
+
+    /**
+     * Test to ensure that the getUrlParam method gets the correct url params set in the url
+     * and formats them as integers, and not as strings.
+     */
+    public function test_geturlparam_returns_set_url_parameters()
+    {
+        $startName = 'start';
+        $countName = 'count';
+        $defaultStart = 0;
+        $defaultCount = 20;
+
+        $expectedStartValue = 1;
+        $expectedCountValue = 5000;
+
+        $getParamWithRandomParams = ['asd' => 'hehe', 'demo' => 'eeeeh', 'start' => '1', 'count' => '5000'];
+        $outputXmlHelper = new OutputXMLHelper();
+        $start = $outputXmlHelper->getUrlParam($startName, $defaultStart, $getParamWithRandomParams);
+        $count = $outputXmlHelper->getUrlParam($countName, $defaultCount, $getParamWithRandomParams);
+
+        $this->assertEquals($expectedStartValue, $start, 'Expected parameter start value does not match the given _GET parameter');
+        $this->assertEquals($expectedCountValue, $count, 'Expected parameter count value does not match the given _GET parameter');
+    }
+
+    /**
+     * Test to ensure that the getUrlParam method gets the default params if _GET param is not set.
+     */
+    public function test_geturlparam_returns_default_if_not_specified()
+    {
+        $startName = 'start';
+        $countName = 'count';
+        $defaultStart = 0;
```

Comment:
Good practice to set these values explicitly instead of getting them from the code, because then if someone changes the default values in the code this test will fail and it will remind me that you may need to modify something external like documentation to reflect the new default values 👍